src/Twig/TemplateEventExtension.php

```diff
@@ -43,24 +39,35 @@ public function getFunctions(): array
 
     /**
      * @param string|string[] $eventName
-     * @param array $context
-     * @param int $cacheTtl
      */
-    public function render(string|array $eventName, array $context = [], int $cacheTtl = 0): string
+    public function render($eventName, array $context = [], int $cacheTtl = -1): string
     {
-        if (is_string($eventName) && empty($cacheTtl)) {
-            $cacheTtl = $this->getTemplateEventCacheTtl($eventName);
+        $dataSerializer = new DataSerializer();
```

Comment:
Hi @MarcoPistorelloAtFiftyDeg , in order to allow other developers to implement their own `DataSerializer` class, it would be nice to inject the `DataSerializerInterface` as a service in the constructor. Thanks!